types/mercadopago/shared/address.d.ts

```diff
@@ -1,7 +1,7 @@
 export interface SimpleAddress {
-  zip_code: string;
-  street_name: string;
-  street_number: string;
+  zip_code?: string | undefined;
+  street_name?: string | undefined;
+  street_number?: number | undefined;
```

Comment:
```suggestion
  zip_code?: string;
  street_name?: string;
  street_number?: number;
```

Reply:
If a user is using TypeScript's `exactOptionalPropertyTypes`, then removing the `| undefined` will prevent them explicitly pass `undefined` for an optional value

Reply:
True